src/plat/morello-qemu/config.cmake

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2023, Hesham Almatary (hesham.almatary@cl.cam.ac.uk)
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(morello-qemu KernelPlatformMorelloQEMU PLAT_MORELLO_QEMU KernelSel4ArchAarch64)
+
+if(KernelPlatformMorelloQEMU)
+    if(NOT ARM_CPU)
```

Comment:
Is `ARM_CPU` used anywhere actually?

Reply:
I think it would be helpful to have some comments at least that there is more to come. Not sure what the the timeline to merge this is -will this remain a patch for some time or do you want some things to be merged?

Reply:
@axel-h Would a GitHub issue be good for the current and future Morello support?

Reply:
@axel-h Ideally baseline platforms (without CHERI support) would be merged soon, with lld support. My local sel4bench numbers for running AArch64 code on a Morello board beat all other seL4's AArch64 platforms by a margin, so this would be useful even without CHERI support. Also my other lld PRs will enable building RISC-V with LLVM/lld

Reply:
I added #1158 that has info of what I am going to submit